pkgs/applications/misc/keepassx/community.nix

```diff
@@ -1,5 +1,5 @@
 { stdenv, fetchFromGitHub, fetchpatch,
-  cmake, libgcrypt, zlib, libmicrohttpd, libXtst, qtbase, qttools, libgpgerror, glibcLocales
+  cmake, libgcrypt, zlib, libmicrohttpd, libXtst, qtbase, qttools, libgpgerror, glibcLocales, libyubikey, libxi, libxtst, qtx11extras
```

Comment:
Hi @Mic92, thanks for your patience with me.  Finally got the LInux build building, but looks like there's some problems with the Mac build, any ideas?